app/models/soda.rb

```diff
@@ -0,0 +1,18 @@
+class Soda < ApplicationRecord
+  belongs_to :creator, class_name: 'User'
+  belongs_to :last_editor, class_name: 'User'
+  belongs_to :country
+  has_many :photos
+  has_and_belongs_to_many :flavors
```

Comment:
I think we should do has_many flavors. And flavors will also has_many sodas. This allow us to use a tag model for flavors. 
Example: You have "strawberry lime lemonade" 
In our db we will have flavors: strawberry, lime, lemonade. When you start typing in str - it brings strawberry and adds it as a tag, then lime, then lemonade. This way this soda will pop up whenever you search for lemonade, lime or strawberry. 
We also can create a field: complete_flavor and store a string that is a sum of those tags in the specific order the user entered it if needed.

This way "apelsin" soda can be tagged as apelsin and also orange since you will see that's it is orange flavor.
I was thinking may be we can create in the notes field: add flavors to soda. As a thought.

The problem with creating self joints and treating flavors and sub flavors is that we will have to curate them. Where the data that "apelsin" is orange will come from? We will have to add this to db and monitor.